dev/implementations/storage_definitions.h

```diff
@@ -126,5 +188,84 @@ namespace sqlite_orm {
             perform_void_exec(db, ss.str());
         }
 
+        template<class... Ts>
+        template<class I>
+        void storage_t<Ts...>::copy_table_from(
+            sqlite3* db,
+            const std::string& tableName,
+            const I& tImpl,
+            const std::vector<const table_xinfo*>& columnsToIgnore) const {  // must ignore generated columns
+            std::stringstream ss;
+            std::vector<std::string> columnNames;
+            tImpl.table.for_each_column([&columnNames, &columnsToIgnore](auto& c) {
+                auto& columnName = c.name;
+                auto columnToIgnoreIt =
+                    std::find_if(columnsToIgnore.begin(), columnsToIgnore.end(), [&columnName](auto tableInfoPointer) {
+                        return columnName == tableInfoPointer->name;
+                    });
+                if(columnToIgnoreIt == columnsToIgnore.end()) {
+                    columnNames.emplace_back(columnName);
+                }
+            });
+            auto columnNamesCount = columnNames.size();
+            ss << "INSERT INTO " << quote_identifier(tImpl.table.name) << " (";
+            for(size_t i = 0; i < columnNamesCount; ++i) {
+                ss << columnNames[i];
+                if(i < columnNamesCount - 1) {
+                    ss << ",";
+                }
+                ss << " ";
+            }
+            ss << ") ";
+            ss << "SELECT ";
+            for(size_t i = 0; i < columnNamesCount; ++i) {
+                ss << columnNames[i];
+                if(i < columnNamesCount - 1) {
+                    ss << ", ";
+                }
+            }
+            ss << " FROM " << quote_identifier(tableName);
+            perform_void_exec(db, ss.str());
+        }
+#endif
+        // jdh

```

Comment:
please don't commit comments saying about the author. We can inspect author of any line using `git blame` instead

Reply:
ok! it's just the way i mark my code where I have done changes so it is easy for me to jump around my changes but I will remove them!!